Cargo.toml

```diff
@@ -1,3 +1,7 @@
+[workspace]
```

Comment:
Reading the doc, it looks like xtask can be used as part of a workspace or not. For this project I think it would be best to not have it as a workspace, so kibi's `Cargo.lock` can stay clean of dev dependencies, and the xtask's one can be used for dev tools.

What do you think, do you mind changing that? It looks like it just requires changing the alias to `run --manifest-path ./xtask/Cargo.toml --` and removing this `[workspace]` section

Reply:
e16f332..f24b462

Reply:
Good point! A workspace sounds good then.

Reply:
1407de8..40755f6